frontend/src/app/modules/team/create-team/create-team.component.ts

```diff
@@ -20,13 +27,20 @@ export class CreateTeamComponent implements OnInit {
                     Validators.required,
                     Validators.minLength(3),
                     Validators.maxLength(50),
+                    Validators.pattern(regexs.teamName)
+                ],
+                [
+                    uniqueTeamNameValidator({} as Team, this.teamService)
```

Comment:
I think it's better to transfer a string there.

Reply:
When I use object, validation works fine, but when use string, after changing name and save, it's trying to validate string, that's equal to current but shouldn't.